spec/spec.yml

```diff
@@ -5232,10 +5391,88 @@ definitions:
       format: integer
     readOnly: true
 
+  SingleBandOptions:
+    type: object
+    properties:
+      band:
+        type: integer
+        description: 'Band number to render'
+      dataType:
+        type: string
+        description: 'Type of data'
+        enum:
+          - DIVERGING
+          - SEQUENTIAL
+          - CATEGORICAL
+      colorBins:
+        type: integer
+        description: |
+          Limit to this number of colors, using interpolation.
+          Defaults to 0, which corresponds to the maximum of 255 (one per normalized pixel value)
+      colorScheme:
+        type: object
+        description: |
+          An array of color hex codes, or an object of breakpoints and their corresponding colors
+          Eg: ["#000000", "#111111", "#222222"]
+          Eg: {"0": "#000000", "10": "#111111", "11": "#222222" }
+      legendOrientation:
+        type: string
+        description: 'Currently unused'
+      extraNoData:
+        description: 'values to treat as NODATA'
```

Comment:
Should probably include a note about how this isn't currently respected and won't be respected outside of categorical color schemes, unless we change that at some point